content/en/dora_metrics/failures/pagerduty.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Configuring PagerDuty for DORA Metrics
+kind: documentation
+description: Learn how to configure PagerDuty as a data source for DORA Metrics incidents.
+is_beta: true
+further_reading:
+- link: "https://app.datadoghq.com/release-notes?category=Software%20Delivery"
+  tag: "Release Notes"
+  text: "Check out the latest Software Delivery releases! (App login required)"
+- link: "/dora_metrics/deployments"
+  tag: "Documentation"
+  text: "Learn about setting up deployments in DORA Metrics"
+- link: "/service_management/events"
+  tag: "Documentation"
+  text: "Learn about Event Management"
+- link: "/monitors/types/metric"
+  tag: "Documentation"
+  text: "Learn about Metric Monitors"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">DORA Metrics is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Overview
+
+[PagerDuty][7] is an incident management platform that equips IT teams with immediate incident visibility, enabling proactive and effective responses to maintain operational stability and resilience.
+
+[PagerDuty incidents][1] can be configured as a data source for failures in DORA Metrics.
+
+## Setup
+
+To integrate your PagerDuty account with DORA Metrics, navigate to **Integrations > Developer Tools** in PagerDuty and click **Generic Webhooks (v3)**. 
+
+1. Click **+ New Webhook** and enter the following details:
+
+   | Variable | Description |
+   |---|---|
+   | Webhook URL | Add `https://webhook-intake.<br>datadoghq.com<br>/api/v2/webhook/`. |
```

Comment:
Could we have this URL dependent on the current site selected? It should be:

`https://webhook-intake.{{< region-param key="dd_site" >}}/api/v2/webhook/`